### PR TITLE
Added error message structure. Initial error is for 404.

### PR DESCRIPTION
**Pull request by dmsakamoto**
Does not affect any other commit

### CODE REVIEW

**Comment by kevhuang** [comment]
Is there a '404' view for `response.render()`?

File: server/components/errors/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  var result = {
+    status: statusCode
+  };
+
+  response.status(result.status);
+  response.render(viewFilePath, function (err) {
+    if (err) {
+      return response.json(result, result.status);
+    }
+    response.render(viewFilePath);
```
Comment (kevhuang) [inline server/components/errors/index.js:19]: Why are you calling `render` again here?

Comment (dmsakamoto) [inline server/components/errors/index.js:19]: Calls a separate view for security

Comment (kevhuang) [inline server/components/errors/index.js:19]: @dmsakamoto but `viewFilePath` doesn't change from line 15 to line 19...? Isn't line 19 doing the same thing as line 15?

**Pull request closed without merging**